be/src/util/frame_of_reference_coding.h

```diff
@@ -37,16 +41,23 @@ namespace doris {
 // 1. Body:
 //      BitPackingFrame * FrameCount
 // 2. Footer:
-//      (2 bit OrderFlag + 6 bit BitWidth) * FrameCount
+//
+//      (8 bit OrderFlag + 8 bit BitWidth) * FrameCount
 //       8 bit FrameValueNum
 //      32 bit ValuesNum
 //
-// The not ascending order BitPackingFrame format:
-//      MinValue, (Value - MinVale) * FrameValueNum
+// if the OrderFlag & 2 == 2: save original values
```

Comment:
why not comment as

OrderFlag 
case 2:
case 1: 
case 0:

Reply:
The lowest bit represents order and the second bit represents if save original value.
---- --00: compressed & not ascending
---- --01: compressed data & ascending
---- --10: original data & not ascending
---- --11: original data & ascending

Reply:
The lowest bit represents order and the second bit represents if save original value.
---- --00: compressed & not ascending
---- --01: compressed data & ascending
---- --10: original data & not ascending
---- --11: original data & ascending

Reply:
> why not comment as
> 
> OrderFlag
> case 2:
> case 1:
> case 0:

+1. which is more clearer.